Tests/Remora.Discord.Tests/Samples/Objects/MESSAGE_REFERENCE/MESSAGE_REFERENCE.json

```diff
@@ -1,4 +1,5 @@
 {
+  "type": 0,
```

Comment:
This field is optional and is missing test data.

Reply:
Hmm, I'm not sure what you'd like me to add here specifically :(

Reply:
I'd like you to add a MESSAGE_REFERENCE.optionals.json file where all optional fields are omitted from the JSON payload. The goal of the *.optional.json files are to test that the data still serializes and deserializes properly when optional fields are omitted from the value.